src/stories/CardNumber.stories.tsx

```
@@ -9,18 +9,46 @@ const meta: Meta<typeof CardNumber> = {
   args: {
     handleChange: () => {},
-    cardNumbers: ["", "", "", ""],
-    errorMessage: ["", "", "", ""],
+    handleStep: () => {},
+    step: 1,
+    cardNumbers,
+    errorMessage,
   },
 };
 
 export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)